Source/app/routes.js

```diff
@@ -74,12 +90,30 @@
         });
     });
 
+
     // =====================================================================
     /////////////////////////////DASHBOARD API//////////////////////////////
     // =====================================================================
     app.post('/api/addDashboard', function(req, res) {
         console.log(req.body);
         var user = req.user;
+
```

Comment:
This does not look right to me? Why is the old code for creating a block in the addDashboard route?